bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/i18n/gree_DE.properties

```diff
@@ -4,90 +4,89 @@ binding.gree.label = GREE Air Conditioner
 binding.gree.description = Dieses Binding integriert Klimaanlagen der Marke GREE
 
 # thing types
-thing-type.gree.airconditioner.label = GREE Klimaanlage
+thing-type.gree.airconditioner.label = GREE 
```

Comment:
revert this change! It's a "Klimaanlage" (one specific thing type, maybe there will be others in the future)